HandheldCompanion/Actions/MouseActions.cs

```diff
@@ -198,9 +200,19 @@ public void Execute(AxisLayout layout, bool touched)
 
             // apply sensitivity, rotation and slider finetune
             deltaVector *= Sensivity * sensitivityFinetune;
-            if (AxisRotated)
-                deltaVector = new(-deltaVector.Y, deltaVector.X);
-            deltaVector *= (AxisInverted ? -1.0f : 1.0f);
+            if (AutoRotate)
+            {
+                SystemManager.DMDO joystickOrientation = SystemManager.GetScreenOrientation();
```

Comment:
I'd prefer SystemManager to inform LayoutManager orientation has changed using an event.
Then LayoutManager would update all Actions once.

Reply:
So like keeping a copy of this in LayoutManager rather than accessing SystemManager's copy of the value?

Reply:
No, I don't want Actions to access external variables. They have to remain autonomous.
You'd need to raise an event from System Manager when orientation changes, get that event captured by LayoutManager for it to pass the information to Actions.

Reply:
Okay, I'll look into this.